tasks/fine_tuning_sbert/src/sentence_transformer.py

```diff
@@ -29,7 +30,8 @@ def fit(self,
             scheduler: str = 'WarmupLinear',
             warmup_steps: int = 10000,
             optimizer_class: Type[Optimizer] = transformers.AdamW,
-            optimizer_params: Dict[str, object] = {'lr': 2e-5, 'correct_bias': True},
+            optimizer_params: Dict[str, object] = {
```

Comment:
We don't need to do this now if it is too much work, but I'd suggest we find a way of including the `lr` parameter in the config file for the sweep... would that work? that way it sets a standard for whenever someone wants to optimize a hyperparameter in this function.

Reply:
Great idea! I made the adjustments to include this for sweeps and for a singular run 😃